docs/source/release_notes/release-notes.rst

```diff
@@ -44,6 +44,20 @@ develop
     *    - Type
          - Change
 
+    *    - |fixed| |userbreak|
+         - TimeLock Server, if configured to use the async lock service, will now throw if a client attempts to start a transaction via the sync lock service.
+           Previously, users which have clients (for the same namespace) running both pre- and post-0.49.0 versions of AtlasDB were able to run transactions against the sync and async lock services concurrently, thus ignoring write-write conflict checks.
```

Comment:
Agreed - this probably makes more sense to an AtlasDB user. Changed.

(Wrote it in this way because if timestamp works properly, then you should not be locking the same timestamp twice, so the main problem is write-write conflicts, if I'm understanding things correctly.)